src/open_inwoner/templates/pages/sitemap/sitemap.html

```diff
@@ -0,0 +1,59 @@
+{% extends "master.html" %}
+{% load i18n list_tags %}
+
+{% block content %}
+    <h1 class="utrecht-heading-1">{% trans "Links naar pagina's op het Open Inwoner Platform" %}</h1>
+
+    <div class="grid">
+        <section>
+            <a class="link" href="{% url 'products:category_list' %}"><h2>{% trans "Onderwerpen" %}</h2></a>
+            <ul>
+                {% for node in category_nodes %}
+                    {% include "pages/sitemap/category_item.html" with node=node %}
+                {% endfor %}
+            </ul>
+        </section>
+
+        {% if request.user.is_authenticated %}
+            <section>
+                {# profile pages heading #}
+                {% for page in cms_profile_pages|slice:":1" %}
+                    {% with link=page.url_name text=page.link_text %}
+                        <a class="link" href="{% url link %}"><h2>{% trans text %}</h2></a>
+                    {% endwith %}
+                {% endfor %}
+                <ul>
+                    {# profile pages #}
+                    {% for page in cms_profile_pages|slice:"1:" %}
+                        {% with link=page.url_name text=page.link_text %}
+                            <li><a class="link" href="{% url link %}">{% trans text %}</h2></a></li>
```

Comment:
The links look nicer with a "link--secondary" class but that could be a future improvement, as a design question.

Reply:
Also this link has a closing H2 tag, but it's missing a starting tag; perhaps no H2 needed at all here?